website/notifications/constants.py

```diff
@@ -10,7 +10,8 @@
     'global_comment_replies': 'Replies to your comments',
     'global_comments': 'Comments added',
     'global_file_updated': 'Files updated',
-    'global_mentions': 'Mentions added'
+    'global_mentions': 'Mentions added',
+    'global_reviews': 'Reviews status change'
```

Comment:
We would need to confirm with Nici but I image this should say "Preprint status change" for now.

Reply:
👍

Reply:
Okay talked with Nici, "Preprint status updated" is the language and submission confirmation email should always be sent regardless of settings.

Reply:
Should that use preprint_word?

Reply:
No, it's a global setting. Changed to "Preprints" instead of "Preprint" though so it's the name of the service.

Reply:
Ah, makes sense 👍

Reply:
Annnnd one more change: "status" --> "submission" because users will get notifications about comment updates.

Reply:
🐼 ok 👍